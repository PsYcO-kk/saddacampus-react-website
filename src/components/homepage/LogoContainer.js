import React from 'react';

const style = {
    "width" : "15%"
};

const LogoContainer = ()=>(
    <div className="container logo-container">
		<img className="img-fluid float-left" style={style} src="./assets/images/brand/16-9.png" id="logo"/>
	</div>
);

export default LogoContainer;
