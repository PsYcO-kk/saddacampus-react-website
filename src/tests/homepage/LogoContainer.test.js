import {shallow} from 'enzyme';
import React from 'react';
import LogoContainer from '../../components/homepage/LogoContainer';

test("should render LogoContainer correctly" , ()=>{
    const wrapper = shallow(<LogoContainer />)
    expect(wrapper).toMatchSnapshot();
})
