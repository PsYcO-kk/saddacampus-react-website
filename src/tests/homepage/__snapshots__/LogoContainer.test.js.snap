// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`testing for Logo container page component 1`] = `
<div
  className="container logo-container"
>
  <img
    className="img-fluid float-left"
    id="logo"
    src="./assets/images/brand/16-9.png"
    style={
      Object {
        "width": "15%",
      }
    }
  />
</div>
`;
